catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/faceting/FacetValueCount.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.operation.faceting;
+
+public class FacetValueCount {
+    private String value;
+    private long count;
+
+    FacetValueCount(String value, long count) {
```

Comment:
Actually, upon looking again, Solr takes in an `int` for this and returns a `long` there's no way to guarantee we don't lose data if we go from `long` to `int` when passing this in, so I think we'll need to use the same mismatched types they do. I don't care for this coupling, but I'm not sure there's a way around it.